src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  function adder(value) {
+    if (arguments.length) {
+      result += value;
+
+      return adder;
+    } else {
```

Comment:
else is redundant here 
```suggestion
    } 
```